### PR TITLE
DOCSUP-20065: Edit and translate to Russian (Window View Functions)

### PR DESCRIPTION
**Pull request by karnevil13**
Changelog category:

- Documentation for #32335 #32402 #32498

### CODE REVIEW

**Comment by kssenii** [comment]
Please see https://github.com/ClickHouse/ClickHouse/pull/32498

File: docs/ru/sql-reference/statements/create/view.md [hunk 1]
```diff
@@ -231,4 +231,120 @@ Code: 60. DB::Exception: Received from localhost:9000. DB::Exception: Table defa
...
+WATCH wv
+```
+
+Когда логи вставлены в таблицу `data`,
+
+``` sql
+INSERT INTO data VALUES(1,now())
+```
+
+запрос `WATCH` должен отразить следующие результаты:
```
Comment (gyuton) [inline docs/ru/sql-reference/statements/create/view.md:327]:
```suggestion
Запрос `WATCH` должен отразить следующие результаты:
```

File: docs/ru/sql-reference/statements/create/view.md [hunk 2]
```diff
@@ -231,4 +231,120 @@ Code: 60. DB::Exception: Received from localhost:9000. DB::Exception: Table defa
...
+
+!!! important "Важно"
+    Это экспериментальная возможность. Ее использование может повлечь потерю совместимости в будущих версиях.
+    Чтобы пользоваться оконными представлениями, и запросами `WATCH`, включите настройку [allow_experimental_window_view](../../../operations/settings/settings.md#allow-experimental-window-view). Введите команду `set allow_experimental_window_view = 1`.
+
+``` sql
+CREATE WINDOW VIEW [IF NOT EXISTS] [db.]table_name [TO [db.]table_name] [ENGINE = engine] [WATERMARK = strategy] [ALLOWED_LATENESS = interval_function] AS SELECT ... GROUP BY time_window_function
+```
+
+Оконное представление может агрегировать данные к временному окну и выводить результаты, когда окно готово к запуску. Оно хранит результаты частичной агрегации во внутренней (или заявленной) таблице, чтобы сократить время задержки, и может отправить результат обработки в заявленную таблицу или отправить уведомления с помощью запроса `WATCH`.
```
Comment (kssenii) [inline docs/ru/sql-reference/statements/create/view.md:244]:
```suggestion
Оконное представление может агрегировать данные к временному окну и выводить результаты, когда окно готово к запуску. Оно хранит результаты частичной агрегации во внутренней (или заявленной) таблице, чтобы сократить время задержки. Результат обработки отправляется в заявленную таблицу и/или в виде уведомления с помощью запроса `WATCH`.
```

**Review by kssenii — CHANGES_REQUESTED** [review]
Ok, but need to merge with master, fix checks and remove accidentally changed files (see there are `contrib/` files added in the diff)

**Pull request closed without merging**